cloner/cmd/clone.go

```diff
@@ -78,20 +84,44 @@ func clone(cmd *cobra.Command, args []string) {
 
 	endpoint, err := transport.NewEndpoint(remote)
 	if err != nil {
-		log.Fatal("failed to parse remote", err)
+		log.Fatal("failed to parse remote: ", err)
 	}
 
 	splittedRepo := strings.FieldsFunc(endpoint.Path, func(c rune) bool { return c == '/' }) // FieldsFunc handles repeated and beginning/ending separator characters more sanely than Split
 	if len(splittedRepo) < 2 {
```

Comment:
Why does this rule exist? Just check if there is a last part, i.e. `len() > 0`.

Reply:
The standard repo URL for providers such as GitHub and GitLab have at least two components:
- namespace or group name (e.g. username, enterprise name, etc.)
- project name

for GitLab, you can also have subgroups.
This check avoids trying to clone wrongly formatted URL. Less ressources and time are wasted.

Reply:
But what if I want to clone a self-hosted git repository at `https://example.org/my-repo.git`? This check seems restrictive AND superfluous. If the URL is wrong, then `git clone` will fail, there is no benefit from not allowing it to fail.

Reply:
I see you point.

The check has been updated